Add venture docs (#3)
crisp-studio/playbook: content/03-ventures/02-pitches.md

# Pitches

> A pitch is **detailed written presentation of a product idea**. If submitted at the right time, pitches will be included and discussed at the next Ideation Breakfast. It is important that the pitches stand alone and need no further explanation. Therefore the pitch should be clearly written and answer the most important questions. Each title is only a working title and can be changed in the actual project.

Reviewer: ```suggestion
A pitch is a **detailed written presentation of a product idea**. If submitted at the right time, pitches will be included and discussed at the next Ideation Breakfast. It is important that the pitches stand alone and need no further explanation. Therefore the pitch should be clearly written and answer the most important questions. Each project title is only a working title and can be changed in the actual project.
```

> The idea for writing pitches originates out of the book "[Thinking Fast and Slow](https://en.wikipedia.org/wiki/Thinking,_Fast_and_Slow)" by Daniel Kahneman. Kahnmann argues that the standard practice of open discussion gives to much weight to opinions of those who speak early and assertively. For us, this is very beneficial to make sure we value the diversity of knowledge and treat different opinions in the same manner.

Reviewer: ```suggestion
The idea for writing pitches originates from the book "[Thinking Fast and Slow](https://en.wikipedia.org/wiki/Thinking,_Fast_and_Slow)" by Daniel Kahneman. Kahneman argues that "the standard practice of open discussion gives too much weight to the opinions of those who speak early and assertively [...]". For us, this is beneficial to make sure we value the diversity of knowledge and treat different opinions equally.
```
